src/Ssh/Ssh/SshCommands/ExportAzSshConfig.cs

```diff
@@ -36,6 +36,14 @@ public override string[] SshArgument
             }
         }
 
+        public override SwitchParameter Rdp
+        {
+            get
+            {
+                return false;
+            }
+        }
```

Comment:
I‘m curious that why this parameter was added into src/Ssh/Ssh/Common/SshBaseCmdlet.cs instead of src/Ssh/Ssh/SshCommands/EnterAzVMCommand.cs. This parameter is not a common parameter, right?

Reply:
@BethanyZhou - Yes, that is not a common parameter. Because both cmdlets have many of the same parameters, I started by adding the common parameters to SshBaseCmdlet.cs and the unique parameters to each cmdlet class. The problem with that is that when using tab completion, the unique parameter names would be suggested first, before the more important mandatory common parameters (such as ResourceGroupName or Name). To avoid that I added all parameters to SshBaseCmdlet.cs, and then suppress the parameters that don't apply to a cmdlet by overriding it as a normal property in the cmdlet class. That is a suggestion I got from PsRemoting cmdlets.